pages/index.vue

```diff
@@ -8,5 +8,12 @@
       Go to example
     </ULink>
   </div>
+</template>
 
-</template>
\ No newline at end of file
+<script lang="ts">
+import { fetchIntro, fetchImageUrl } from "~/api/wikipediaSource";
+const s : string = await fetchIntro("Jupiter");
+console.log(s)
+const url : string = await fetchImageUrl("Jupiter", 100);
```

Comment:
`await` not necessary here
```suggestion
const url : string = fetchImageUrl("Jupiter", 100);
```

Reply:
Same answer as above.